lib/modules/manager/npm/post-update/npm.ts

```diff
@@ -196,6 +196,20 @@ export async function generateLockFile(
       } /* v8 ignore stop -- needs test */
     }
 
+    if (postUpdateOptions?.includes('npmInstallTwice') === true) {
+      logger.debug('Running npm install twice');
+      // Run the install command twice to ensure the lock file is up to date
+      // iterate through commands and if any command starts with `npm install`, add it again
+      const existingCommands = [...commands];
+      commands = [];
```

Comment:
Instead of recreating you can use [`splice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)